services/workflows-service/prisma/schema.prisma

```diff
@@ -95,11 +95,18 @@ model EndUser {
   @@index([projectId])
 }
 
+enum BusinessPosition {
```

Comment:
```suggestion
enum PositionInBusiness {
```
```suggestion
enum BusinessPosition {
```

Reply:
If you change the column name to `position`, then you can ignore this comment. I suggested it just to be consistent with the column name.